.binstar.yml

```diff
@@ -6,7 +6,6 @@ platform:
   - win-64
   - win-32
 engine:
-  - python=2.6
```

Comment:
I'm also entirely happy to drop Python 2.6 if you think we've reached that
point.  My only request is that we ask for input for a suitable period
before dropping it.

On Mon, Aug 22, 2016 at 11:09 AM, Erik Welch <notifications@github.com>
wrote:

> In .binstar.yml
> <https://github.com/pytoolz/toolz/pull/326#discussion_r75695542>:
>
> > @@ -6,7 +6,6 @@ platform:
> >    - win-64
> >    - win-32
> >  engine:
> > -  - python=2.6
>
> Fair 'nuff. Re-added Python 2.6. It wasn't that painful--this time.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pytoolz/toolz/pull/326/files/0ddc67ff372b46ab8c7a406f05809f016008b9a4#r75695542>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASszJ6fvs1uTHpPEUZbx88lgf_F36SZks5qibuhgaJpZM4JpYoz>
> .
>